src/Mod/TechDraw/Gui/CommandExtensionPack.cpp

```diff
@@ -458,8 +458,8 @@ CmdTechDrawExtensionThreadBoltSide::CmdTechDrawExtensionThreadBoltSide()
 {
     sAppModule = "TechDraw";
     sGroup = QT_TR_NOOP("TechDraw");
-    sMenuText = QT_TR_NOOP("Add Cosmetic Thread Bolt Side View");
-    sToolTipText = QT_TR_NOOP("Add a cosmetic thread to the side view of a bolt/screw/rod:<br>\
+    sMenuText = QT_TR_NOOP("Add cosmetic thread bolt side view");
+    sToolTipText = QT_TR_NOOP("Adds a cosmetic thread to the side view of a bolt/screw/rod:<br>\
```

Comment:
```suggestion
    sToolTipText = QT_TR_NOOP("Adds a cosmetic thread to the side view of a bolt/screw/rod between two selected parallel lines")
```
then remove next 3 lines